psutil/_psutil_osx.c

```diff
@@ -860,6 +874,48 @@ psutil_boot_time(PyObject *self, PyObject *args) {
     return Py_BuildValue("f", (float)boot_time);
 }
 
+/*
+ * Return a Python float indicating the value of the temperature
+ * measured by an SMC key
+ */
+static PyObject *
+smc_get_temperatrue(PyObject *self, PyObject *args) {
+	char* key;
+	float temp;
+    if (! PyArg_ParseTuple(args, "s", &key)) {
+        return NULL;
+	}
+	temp = SMCGetTemperature(key);
+    return Py_BuildValue("f", (float)temp);
+}
+
+/*
+ * Return a Python float indicating the number of fans in the system
+ */
+static PyObject *
+smc_get_fan_num(PyObject *self, PyObject *args) {
```

Comment:
please rename this to `psutil_fans_count` so that we are consistent with `psutil_cpu_count_logical` and `psutil_cpu_count_physical`